app/javascript/components/upload/UploadWizard.js

```diff
@@ -179,43 +181,45 @@ export default function UploadWizard({ studyAccession, name }) {
     })
   }, [studyAccession])
 
-  return <div className="">
-    <div className="row padded">
-      <div className="col-md-10">
-        <h4>{studyAccession}: {name}</h4>
+  return <UserProvider>
+    <div className="">
+      <div className="row padded">
+        <div className="col-md-10">
+          <h4>{studyAccession}: {name}</h4>
+        </div>
+        <div className="col-md-2">
+          <a href={`/single_cell/study/${studyAccession}`}>View Study</a>
```

Comment:
This link could use styling as a Bootstrap button - it is very easy to miss without it.

Reply:
This is addressed in the later (but now-merged) UX refinement PR